frontend/apps/reader/modules/readerzooming.lua

```diff
@@ -208,8 +212,14 @@ function ReaderZooming:_updateConfigurable(zoom_mode)
 
     local zoom_mode_genus, zoom_mode_type = self:mode_to_combo(zoom_mode)
 
+    -- FIXME(ogkevin): when zoom_mode is "free", zoom_mode_genus is nil
+    -- This is because in the mode_to_combo maping, free doesn't exsit.
+    -- Manual does, but is free and manual the same thing?
+    logger.dbg("ReaderZooming:_updateConfigurable", zoom_mode, zoom_mode_genus, zoom_mode_type)
+
     -- Configurable keys aren't prefixed, unlike the actual settings...
-    configurable.zoom_mode_genus = zoom_mode_genus
+    -- TODO(ogkevin): hack for nil zoom_mode_genus, needs confirmation if accaptable
+    configurable.zoom_mode_genus = zoom_mode_genus and zoom_mode_genus or 0
```

Comment:
@Frenzie This is also broken on master 🙈 
This seems to fix it, but I'm not sure if "free" and "manual" conceptually are the same thing. Any ideas?

Reply:
I doubt it. Manual has odd/even and is meant to cut off margins. Free would imply something more like how you'd read a document on a phone, wouldn't it? (Currently missing really.)

Reply:
I see, I can't seem to be able to trigger "Free" zoom on a touch device, indeed.

![image](https://github.com/user-attachments/assets/ed0b1ac3-11b4-4605-a8d9-70db48e9a382)

I'll then leave my hack to set it to manual for now to just prevent the crash. I'll leave a unassigned fixme note as well.
Shall I open another issue to track this, or is the fixme enough?
 
Most likely a proper fix would be to fully implement free zoom/movement indeed.

Reply:
> Shall I open another issue to track this, or is the fixme enough?

https://github.com/koreader/koreader/issues/5524

(but also if by tracking you mean something other than discussing ideas then a todo is more than enough)